src/daft-logical-plan/src/builder.rs

```diff
@@ -188,11 +188,19 @@ impl LogicalPlanBuilder {
     }
 
     pub fn select(&self, to_select: Vec<ExprRef>) -> DaftResult<Self> {
+        let expr_resolver = ExprResolver::builder().allow_actor_pool_udf(true).build();
```

Comment:
Out of curiosity, why does each (or some) method create their own expr_resolver? Maybe the builder could hold a single expr_resolver for its schema, then `resolve_single` takes a single arg and closes over self.schema().

But there may be more to this I am not familiar with yet, just curious.

Reply:
Each logical op may resolve expressions differently. For example, Agg expects expressions to be aggregation expressions, whereas Project expects no aggregation expressions. Take a look at the parameters to the ExprResolver builder!